vllm/v1/attention/backends/utils.py

```diff
@@ -64,6 +64,14 @@ class CommonAttentionMetadata:
 
     causal: bool = True
 
+    # Encoder/cross-attention specific fields (optional)
+    encoder_seq_start_loc: Optional[torch.Tensor] = None
```

Comment:
I don't think this is being used anywhere?

Reply:
removed!